chain/filecoin/client.go

```diff
@@ -100,6 +103,9 @@ func (client *Client) Tx(ctx context.Context, txID pack.Bytes) (account.Tx, pack
 	if messageLookup == nil {
 		return nil, pack.NewU64(0), fmt.Errorf("searching state for txid %v: not found", msgID)
 	}
+	if messageLookup.Receipt.ExitCode.IsError() {
+		return nil, pack.NewU64(0), fmt.Errorf("transaction execution error: %v", messageLookup.Receipt.ExitCode.String())
```

Comment:
```suggestion
		return nil, pack.NewU64(0), fmt.Errorf("executing transaction: %v", messageLookup.Receipt.ExitCode)
```
You do not need to call `String()` on a value if it is being formatted with `%v` (that is the point of `%v`).